tools/elf2efi.py

```diff
@@ -508,12 +508,15 @@ def elf2efi(args: argparse.Namespace):
     opt.SizeOfImage = align_to(
         sections[-1].VirtualAddress + sections[-1].VirtualSize, SECTION_ALIGNMENT
     )
-    opt.SizeOfHeaders = align_to(
-        PE_OFFSET
-        + coff.SizeOfOptionalHeader
-        + sizeof(PeSection) * coff.NumberOfSections,
-        FILE_ALIGNMENT,
-    )
+
+    SizeOfHeaders = PE_OFFSET + coff.SizeOfOptionalHeader + sizeof(PeSection) * coff.NumberOfSections
+
+    # Minimum required for a PE addon with a cmdline, sbat, uname and osrel sections.
+    if SizeOfHeaders < 0x400:
+        SizeOfHeaders = 0x400
```

Comment:
where precisely is this name coming from?

Reply:
or conversely, why not update `coff.NumberOfSections` to the right value for add-ons? why specify something literal manually here?